wled00/wled.cpp

```diff
@@ -495,6 +495,16 @@ void WLED::setup()
   initServer();
   DEBUG_PRINT(F("heap ")); DEBUG_PRINTLN(ESP.getFreeHeap());
 
+  // Seed FastLED random functions with an esp random value, which already works properly at this point.
+#if defined(ARDUINO_ARCH_ESP32)
+  const uint32_t seed32 = esp_random();
```

Comment:
minor: please check if the header file for esp_random is already included.
```C++
#if defined(ARDUINO_ARCH_ESP32)
#include "esp_random.h"
#endif
```

Reply:
Should I also include something for the esp8266?